src/iris/ui/templates/plan.html

```diff
@@ -255,6 +258,13 @@ <h4><i class="icon-plus icon">+</i> Add application tracking template</h4>
           <div class="notification-header">HTML</div>
           <pre class="notification-body">{{breakLines email_html}}</pre>
         </div>
+        {{/isEqual}}
+        {{^isEqual ../tracking_type 'email'}}
```

Comment:
I think this should be a # not ^ ?

Reply:
While I cannot point to documentation - I couldn't find any. I found a comment on stackoverflow (?) mentioning this usage negates the helpers behavior. I've tested this and it works as intended as far as I can tell.
Also see: https://github.com/linkedin/oncall/pull/128#discussion_r160308027